src/olympia/abuse/management/commands/sync_cinder_policies.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia.abuse.tasks import sync_cinder_policies
+
+
+class Command(BaseCommand):
+    help = (
+        'Sync all cinder policies to our local database. Syntax: \n'
+        '    ./manage.py sync_cinder_policies'
+    )
+    log = olympia.core.logger.getLogger('z.abuse')
+
+    def handle(self, *args, **options):
+        sync_cinder_policies.apply()
```

Comment:
`apply()` will run the task locally synchronously, not through a celery worker. Was that your intention ? We typically use `.delay()` or `.apply_async()` to queue the task.

Reply:
Good catch. Only had it that way for local testing.

Reply:
Local envs should have a working celery worker (tests don't, but they run with `CELERY_TASK_ALWAYS_EAGER = True`)

Reply:
if this is only run as a command/cron we don't really need the task anyway, imo - the request should complete quickly and there would be immediate feedback in the console/logs.

Reply:
(just an observation - I'm not requesting a change)

Reply:
I think you can argue either side. Why I chose a task in this case:
- I typically prefer retry able mechanisms for things that depend on external APIs as they can randomly fail. (I should actually add a retry on non 401/3 network errors
- this is work that can run asynchronously as no one directly waits for it to finish and putting it in a task emphasizes that mental model.

Where would you put it outside of a task out of curiosity? In a util method or somewhere specific? I'm assuming we want to either trigger this via management command or via cron so it cannot be implemented directly as either.

Reply:
Either a function in utils that the management command calls, or directly in the management `handle` method.  Or in `cron.py` - if the function is defined in `settings.CRON_JOBS` it is callable via `./manage.py cron xxx`.  

But it's fine as it is.

Reply:
TIL. Cool.